### DISCUSSION TITLE
1.4.0

### DISCUSSION THREAD

**Discussion by kraibse** [kraibse/obsidian-table-sorting#19] [General]
# Experimental build - pre-1.4

## What's Changed
**This plugin finally works with the new tables that came with the Obsidian update 1.5.x**

* I've reworked the plugin to be more mobile friendly #17 (Any testing on that regard is appreciated)
* Upgraded preview videos by @kraibse in https://github.com/kraibse/obsidian-table-sorting/pull/14
* Removed obsolete files && upgraded gif preview by @kraibse in https://github.com/kraibse/obsidian-table-sorting/pull/15
* Updated README.md for better clarity by @kraibse in https://github.com/kraibse/obsidian-table-sorting/pull/16

**Full Changelog**: https://github.com/kraibse/obsidian-table-sorting/compare/master...version-v1.4-upgrade

## How can you use the new sorting system?

> In the coming updates I will update the README accordingly

**Open the context menu for a column**
* (desktop) - right click anywhere within a column
* (mobile) - hold the drag handle of a column

**Use any of the following options**

![image](https://github.com/kraibse/obsidian-table-sorting/assets/33869366/eea8421e-27d8-4988-bc06-f62100f61035)

* **Single column mode** can be simply done by selecting either of the sorting modes
* For **multi column mode** your columns need to selected first, then you can sort
* To apply the original order, simply use the reset filters option

## Known issues

There still are some things I want to get done during the 1.4.x updates. This includes:
- polishing the icons in the dropdown
- fixing some issues with additional table dropdown
- code maintenance

Gladly open an issue if you find something notable.

## What's Changed
* Upgraded preview videos by @kraibse in https://github.com/kraibse/obsidian-table-sorting/pull/14
* Removed obsolete files && upgraded gif preview by @kraibse in https://github.com/kraibse/obsidian-table-sorting/pull/15
* Updated README.md for better clarity by @kraibse in https://github.com/kraibse/obsidian-table-sorting/pull/16
* v1.4 upgrade  by @kraibse in https://github.com/kraibse/obsidian-table-sorting/pull/18

**Full Changelog**: https://github.com/kraibse/obsidian-table-sorting/compare/1.3.0...1.4

<hr /><em>This discussion was created from the release <a href='https://github.com/kraibse/obsidian-table-sorting/releases/tag/1.4'>1.4.0</a>.</em>

**Comment by growdigital** [comment]
this is a thing of beauty, thank you!
